WordsGame/Utils.cs

```diff
@@ -23,4 +60,47 @@ public static string PickRandom(List<string> entries)
         var i = new Random().NextInt64() % entries.Count;
         return entries[(int)i];
     }
+
+    public static int MapCharToPoints(char letter)
+    {
+        var upperCaseLetter = char.ToUpper(letter);
+        switch (upperCaseLetter)
+        {
+            case 'D':
+            case 'G':
+                return 2;
+            case 'B':
+            case 'C':
+            case 'M':
+            case 'P':
+                return 3;
+            case 'F':
+            case 'H':
+            case 'V':
+            case 'W':
+            case 'Y':
+                return 4;    
+            case 'K':
+                return 5;   
+            case 'J':
+            case 'X':
+                return 8;  
+            case 'Q':
+            case 'Z':
+                return 10;   
+            default:
+                return 1;
+        }
```

Comment:
Sehr gut, eine der wenigen Fällen, in denen das Fallthrough-Verhalten von switch/case nützlich ist!